client/settings/fraud-protection/advanced-settings/index.js

```diff
@@ -146,6 +188,45 @@ const FraudProtectionAdvancedSettingsPage = () => {
 		}
 	}, [] );
 
+	// Intersection observer callback for tracking card viewed events.
+	const observerCallback = ( entries ) => {
+		entries.forEach( ( entry ) => {
+			const { target, intersectionRatio } = entry;
+
+			if ( 0 < intersectionRatio ) {
+				// element is at least partially visible.
+				const { id } = target;
+				const { event } = observerMapping[ id ] || {};
+
+				if ( event ) {
+					wcpayTracks.recordEvent( event );
+				}
+
+				cardObserver.current?.unobserve(
+					document.getElementById( target.id )
```

Comment:
You already have `id` destructured from `target` above, we can just use `id` here.

Reply:
Yes. I was supposed to change that after Eduardo and I paired up to work on this. Thanks for catching it.

Reply:
Thanks for catching this. I had mentioned it when Eduardo and I paired up on this, but I forgot to circle back to update it. Fixed in [498643a](https://github.com/Automattic/woocommerce-payments/pull/5820/commits/498643a79759ac936ecc33e22f2e96ae6eb7a2e0).